pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -103,6 +103,7 @@ mkCommon type rec {
     mv $out/ThirdPartyNotices.txt $out/share/doc/$pname/$version/
 
     ln -s $out/dotnet $out/bin/dotnet
+    wrapProgram "$out/bin/dotnet" --set AZURE_TELEMTRY_DISABLE 1
```

Comment:
Typo: TELEMTRY should be TELEMETRY

Reply:
I swear i built this and it worked...
Maybe i had the environment variable set from another shell?

Reply:
I fixed the typo, but I can't seem to replicate the original issue anymore with dotnet from nixpkgs, or even the nixpkgs versions that I was having issues with earlier, which probably means the issue still wasn't fixed, but there's some impurity in my system causing this to happen. I tried clearing all the dotnet caches I could find though.
 If someone else having issues could make sure this fixes the problem, that would be great. Otherwise I would recommend still merging this regardless.